### PR TITLE
[1.0] Stabalize packages for 1.0.5 servicing release

### PR DESCRIPTION
**Pull request by weshaggard**
cc @gkhanna79 @leecow @ericstj @eerhardt

### CODE REVIEW

**Comment by weshaggard** [comment]
>Before merging, let's wait until we get one successful build of core-setup with the updated package versions to ensure everything is working properly.

Yep that is why I marked these PR's as no-merge. I just wanted to have them ready.

**Comment by PinpointTownes** [comment]
Really great news!

Question: will the updated `System.Net.Http` 4.1.2 LTS package fix these two issues?
  - https://github.com/dotnet/corefx/issues/10047
  - https://github.com/dotnet/corefx/issues/11100

**Comment by weshaggard** [comment]
@PinpointTownes - @karelz @davidsh can correct me if I'm wrong but I believe #11100 was already fixed with https://www.nuget.org/packages/System.Net.Http/4.3.1. I don't think we have a solution for #10047 yet so I don't believe it is fixed.

**Comment by PinpointTownes** [comment]
@weshaggard sure, but isn't 4.3.1 part of .NET Core 1.1? (i.e, not LTS)

Concerning https://github.com/dotnet/corefx/issues/10047, that's unfortunate because it's a very annoying bug that causes exceptions at runtime if you don't manually add the binding redirects.

**Comment by weshaggard** [comment]
>@weshaggard sure, but isn't 4.3.1 part of .NET Core 1.1? (i.e, not LTS)

Sorry you are correct 4.3.1 isn't part of .NET Core 1.0 LTS. However the fix for that issue isn't about .NET Core LTS it is about .NET Framework so I don't think there are any plans to port it to the LTS branch.

**Comment by PinpointTownes** [comment]
@weshaggard true, but as a library author, I have packages that offer both .NET Desktop and .NET Core (or .NET Standard) TFMs and depend on System.Net.Http. Since I target the LTS version, the users of my packages using them in their .NET Desktop apps have to manually reference `System.Net.Http` 4.3.1 instead of the LTS version to work around this bug. The experience would be much better if they didn't have to do that, no?

**Comment by weshaggard** [comment]
>@weshaggard true, but as a library author, I have packages that offer both .NET Desktop and .NET Core (or .NET Standard) 

If you are cross-compiling for both .NET Framework and .NET Core then I would suggest you also have a unique set of dependencies for each in your library package and for .NET Framework you can reference the 4.3.1 package. While that isn't the greatest solution it is probably the best option, as it is hard to put every fix we make into the LTS packages. LTS by definition is supposed to only take critical fixes to ensure high quality for the long term.

**Comment by PinpointTownes** [comment]
> While that isn't the greatest solution it is probably the best option, as it is hard to put every fix we make into the LTS packages.

Thanks for the workaround (TBH, I'm not sure it's better than forcing my users to manually reference the FTS bits) 

> LTS by definition is supposed to only take critical fixes to ensure high quality for the long term.

I definitely understand that, but I would be tempted to say that a package that doesn't work (at all) on >.NET 4.6 can be considered as a critical bug that qualifies for a LTS fix.

**Comment by weshaggard** [comment]
>I definitely understand that, but I would be tempted to say that a package that doesn't work (at all) on >.NET 4.6 can be considered as a critical bug that qualifies for a LTS fix.

I only know parts of that particular issue but it is my understanding that it is only certain things that don't work. The majority of scenarios do work it is only a few that cause trouble.

**Comment by PinpointTownes** [comment]
I guess I'm not very lucky then, because I saw the `Inheritance security rules violated by type: 'System.Net.Http.WebRequestHandler'. Derived types must either match the security accessibility of the base type or be less accessible.` exception is most ASP.NET/ASP.NET Core apps I've moved from .NET 4.5.2 to >.NET 4.6.

Anyway, thanks for your answer! (FWIW, I really appreciate your openness)

File: pkg/Microsoft.Private.PackageBaseline/stable.packages.props [hunk 1]
```diff
@@ -1356,5 +1356,49 @@
...
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.debian.8-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.fedora.23-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.native.System.Security.Cryptography">
+      <Version>4.0.1</Version>
```
Comment (bartonjs) [inline pkg/Microsoft.Private.PackageBaseline/stable.packages.props]: On the one hand, I'm confused as to why the version for the wrapper package is .1 when the individual ones are .2.  But on the other hand, I'll believe it's correct.

**Comment by davidsh** [comment]
>@PinpointTownes - @karelz @davidsh can correct me if I'm wrong but I believe #11100 was already fixed with https://www.nuget.org/packages/System.Net.Http/4.3.1. I don't think we have a solution for #10047 yet so I don't believe it is fixed.

Yes, #11100 was fixed via the System.Net.Http NuGet package version 4.3.1.

File: pkg/Microsoft.Private.PackageBaseline/stable.packages.props (hunk 1 above)
Comment (weshaggard) [inline pkg/Microsoft.Private.PackageBaseline/stable.packages.props]: The others are 1.0.2 and this one is 4.0.1. This is correct, I also did a double take. It has to do with the fact that we versioned the shim packages at 1.0.0 at first and in .NET Core 1.1 we moved them all to 4.3.0 as well.

**Pull request merged**